README.md

```diff
@@ -109,6 +133,11 @@ To get our testing framework together, a lot of research was needed to get Jest
 * Sessions can end and then rollover into another with the same host and guests.
 
 ### Code refactorings
+
+* Refactored SQL strings within the controller to use pg-format, a library which uses C language style delimeters in constructing SQL query strings to thwart SQL injection attacks.
```

Comment:
FYI, you do still need to go and fix those up. They are still backwards. Happy to show you an example at some point of what I mean.